templates/index.html

```diff
@@ -272,25 +272,7 @@ <h4 class="title"></h4>
             <p class="mainStreet" style="margin:0"></p>
             <p class="secondaryStreet" style="margin:0"></p>
             <p class="junction" style="margin:0"></p>
-            <p class="involved_type" style="margin:0"></p>
```

Comment:
Why did you remove these? Isn't it better to leave the placeholders so the order can be controlled by editing the HTML and not the code? If the data isn't there, they'll just stay empty.